v2/tools/asoctl/cmd/crd_clean.go

```diff
@@ -13,18 +13,25 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client/config"
 
 	"github.com/Azure/azure-service-operator/v2/api"
+
 	"github.com/Azure/azure-service-operator/v2/tools/asoctl/internal/crd"
 )
 
 // newCRDCleanCommand creates a new cobra command for cleaning deprecated CRDs from a cluster
 func newCRDCleanCommand() *cobra.Command {
 	dryRunFlagName := "dry-run"
+	dryRun := false
 
 	cmd := &cobra.Command{
 		Use:   "clean",
 		Short: "Clean deprecated CRD versions from cluster",
+		Args:  cobra.NoArgs,
 		RunE: func(cmd *cobra.Command, args []string) error {
-			cfg := config.GetConfigOrDie()
+			cfg, err := config.GetConfig()
+			if err != nil {
+				return errors.Wrap(err, "unable to get kubernetes config")
```

Comment:
```suggestion
				return errors.Wrap(err, "unable to get Kubernetes config")
```

Reply:
I think it's OK to cap in the err just not the first letter (why not who knows?)

Reply:
Fixed in #2867